scripts/genie.lua

```diff
@@ -313,6 +313,7 @@ function exampleProjectDefaults()
 	configuration { "mingw*" }
 		targetextension ".exe"
 		links {
+			"comdlg32",
```

Comment:
Why is this not needed by GCC?

Reply:
I am not sure why this is needed for clang only. But this fixes the following linking error:

```
Linking example-17-drawstress
ld.lld: error: undefined symbol: GetOpenFileNameA
>>> referenced by ../../../examples/common/entry/dialog.cpp:223
>>>               libexample-commonRelease.a(dialog.o):(openFileSelectionDialog(bx::FilePath&, FileSelectionDialogType::Enum, bx::StringView const&, bx::StringView const&))
```